### ISSUE TITLE
Summarization of the technical replicate

### ISSUE THREAD

**Issue by Laurems4ms** [Nesvilab/FragPipe-Analyst-doc#63]
Hey the FragPipe Team !
Thank you very much for this tool, it is very easy to use !
I am just wondering if i am doing well, I have two strange thing for me... 
I do an DIA analysis, with median centred normalization, no imputation, Min percentage of non-missing values in at least one condition 50%. I have 3 conditions some with 1, or 2 or 3 or 4 or 5 technical replicate and other without.
The matrix that I obtained after the analysis with FragPipeAnalyst are always with technical replicates separated, I means there is no summarization of the technical replicate in the Normalized_matrix, Filtered_matrix or full_dataset.
 For the limma GC it is also like that ? 
Is it normal ? Or it is an issu with my experiment_annotation file ?
![image](https://github.com/user-attachments/assets/adc2e8bf-96a5-49ae-9022-5cfc1f979da8)

Also the normalized_matrix and the full_dataset show positive and negative numbers between -5 and 9...
![image](https://github.com/user-attachments/assets/fd7beffc-89a7-420a-bc2f-513aa4c59c00)

Have you any advice ? Thank you very much for your help !

Laure.

**Comment by hsiaoyi0504** [comment]
> There is no summarization of the technical replicate in the Normalized_matrix, Filtered_matrix or full_dataset. For the limma GC it is also like that ? Is it normal ? Or it is an issu with my experiment_annotation file ?

Yes, currently we treated every sample separately, which means technology replicates are not summarized. The fundamental reason is that replicates column could be biological replicates as well some times, so we don't have definite answer to summarize replicates or not, but it could potentially to have other options in the future.

> Also the normalized_matrix and the full_dataset show positive and negative numbers between -5 and 9...
It's log 2 transformed, so it seems to be ok. But as always, you need to make sure the imputation makes sense without introducing artificial things.
